i18n/ko.json

```diff
@@ -225,17 +225,17 @@
     "tabView": "보기 설정",
     "thumbnailHeading": "섬네일 보기",
     "variousSettings": "기타 설정",
-    "viewSettings": "더 많은 보기",
-    "wordCloud": "워드 클라우드 & 최근 열린 파일",
+    "viewSettings": "기타 보기",
+    "wordCloud": "워드 클라우드 & 최근에 열었던 파일",
     "zoom": "확대 / 축소"
   },
   "SHORTCUTS": {
-    "andClickAnyFile": "파일을 클릭하여 자세히 보기",
-    "andPress": "누르기",
+    "andClickAnyFile": "누른 상태에서 파일을 클릭하여 자세히 보기",
+    "andPress": "누른 상태에서 누르기:",
     "focusOnFile": "파일 검색에 입력",
     "focusOnFuzzy": "퍼지 검색에 입력",
     "focusOnMagic": "마법 검색에 입력",
-    "hold": "Hold",
+    "hold": "",
```

Comment:
238 + 233 ->233 Hold and click any file to see the file details
238 Is there no problem even if it's blank?